### ISSUE TITLE
[BUG] Log output with a newline on the end never gets flushed.

### ISSUE THREAD

**Issue by brandonbloom** [deref/exo#97]
**Describe the bug**

**To Reproduce**

**Expected behavior**

**Screenshots**

**System Info (please complete the following information):**
 - OS: [e.g. macOS]
 - Component [e.g. gui, cli]
 - Version [e.g. 2021.08.04_1]

**Additional context**

**Comment by brandonbloom** [comment]
@kendru seems like you were auto-assigned, is that as part of the template ... for triage?

**Comment by kendru** [comment]
@brandonbloom Yeah - I just had it do that so that I could triage them. Once we start getting some tickets opened by the community, I figured that we could figure out how to better handle them at that point.
